docs/source/chatops/aliases.rst

```diff
@@ -133,6 +133,13 @@ In this the action google.get_search_results would be called with the parameters
        query: StackStorm
        limit: 10
 
+Additional ChatOps parameters passed to the command
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+An execution triggered via chatops will contain variables such as ``action_context.api_user``, ``action_context.user`` and ``action_context.source_channel``. ``api_user`` is the user who's kicking off the chatops command from
```

Comment:
Since this is already documented in docs/source/actions.rst it might be better to just link there or maybe refactor it in a single file and use an include directive :)

I'm all for making it user-friendly and repeating it as much as needed, but if it's repeated "manually" (e.g. not auto generating it and / or using include directive) it's easy for such information to get out of sync with the code (and documentation which is incorrect is usually worse then no documentation).

One improvement we could do in the future is to add some documentation for each attribute in the code and then auto-generate a table with each attribute description and include it in the documentation (we already do this for runner parameters, RBAC permission types, etc.).